awx/main/models/ha.py

```diff
@@ -54,6 +54,11 @@ def has_policy_changes(self):
         return self._values_have_edits(new_values)
 
 
+class InstanceLink(BaseModel):
+    source = models.ForeignKey('Instance', on_delete=models.CASCADE, related_name='+')
+    target = models.ForeignKey('Instance', on_delete=models.CASCADE, related_name='reverse_peers')
```

Comment:
So we have 2 relations, `instance.peers` and `instance.reverse_peers`? I think I read that right, and that looks good.

Reply:
Yep, you got it.